### PR TITLE
DX-1302 TT-8812 TT-11522 configure tyk-oss to create a Kubernetes Secret for Tyk Operator

### PR DESCRIPTION
**Pull request by carolwkc**
### For internal users - Please add a Jira DX PR ticket to the subject!
<br>

[DX-1302](https://tyktech.atlassian.net/browse/DX-1302)

---

### Preview Link

<br>

[Preview](https://deploy-preview-4540--tyk-docs.netlify.app/docs/product-stack/tyk-charts/tyk-oss-chart/#create-a-kubernetes-secret-for-tyk-operator)

### Description

<br>

#### Screenshots (if appropriate)
<br>

## Checklist

- [ ] I have added a **preview link** to the PR description.
- [ ] I have [reviewed the guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING.md) for contributing to this repository.
- [ ] I have [read the technical guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING-TECHNICAL-GUIDE.md) for contributing to this repository.
- [ ] Make sure you have started *your change* off *our latest `master`*.
- [ ] I **labelled** the PR

[DX-1302]: https://tyktech.atlassian.net/browse/DX-1302?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: tyk-docs/content/product-stack/tyk-charts/tyk-oss-chart.md [hunk 1]
```diff
@@ -183,6 +183,21 @@ global:
        keyName: "redisPassKey"
 ```
 
+### Create secret for Tyk Operator
```
Comment (buraksekili) [inline tyk-docs/content/product-stack/tyk-charts/tyk-oss-chart.md]: What do you think?
```suggestion
### Create a Kubernetes Secret for Tyk Operator
```

File: tyk-docs/content/product-stack/tyk-charts/tyk-oss-chart.md [hunk 2]
```diff
@@ -183,6 +183,21 @@ global:
...
+
+```yaml
+# operatorSecret controls if a secret needed to connect to Operator will be created
+operatorSecret:
+  # enabled if set to true creates secret
+  enabled: true
+  # OSS doesn't have concept of OrgID. But we need to support some features (eg. basic auth key) in OSS
+  # You can set it to any arbitary value
+  orgID: "orgid"
+
```
Comment (buraksekili) [inline tyk-docs/content/product-stack/tyk-charts/tyk-oss-chart.md]: we can remove redundant line
```suggestion
```

**Review by buraksekili — APPROVED** [review]

**Review by dcs3spp — APPROVED** [review]
PR LGTM once peer approved by @komalsukhani, I will merge and release.

**Comment by carolwkc** [comment]
Thanks @dcs3spp . Komal is on leave and Burak has helped to review. Please release this when Tyk Charts 1.4.0 is released later this week.

**Pull request merged**